<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stopmotion_UI</class>
 <widget class="QDialog" name="Stopmotion_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>778</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="12">
    <widget class="QFrame" name="video_preview">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="live_button">
     <property name="toolTip">
      <string>Live view</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="frameoverlay_button">
     <property name="toolTip">
      <string>Overlay last frame</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="preview_button">
     <property name="toolTip">
      <string>Preview sequence</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="KLineEdit" name="sequence_name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Sequence name</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QSpinBox" name="frame_number">
     <property name="toolTip">
      <string>Frame number</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="11">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QToolButton" name="capture_button">
     <property name="toolTip">
      <string>Capture frame</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QToolButton" name="removelast_button">
     <property name="toolTip">
      <string>Remove current frame</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="9">
    <widget class="KComboBox" name="capture_device">
     <property name="toolTip">
      <string>Capture device</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="10">
    <widget class="QPushButton" name="button_addsequence">
     <property name="text">
      <string>Add sequence to project</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Stopmotion_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Stopmotion_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
